Makefile

```diff
@@ -84,14 +84,16 @@ dev-build: $(PROJECT_PATH)/Dockerfile
 		$(PROJECT_PATH)
 
 .PHONY: ci-build
-ci-build: export APISONATOR_REL?=v$(shell ruby -r$(PROJECT_PATH)/lib/3scale/backend/version -e "puts ThreeScale::Backend::VERSION")
+ci-build: export APISONATOR_REL?=v$(shell docker run --rm -e RBENV_VERSION='2.3.6' -v $(PROJECT_PATH):/tmp/apisonator:z \
```

Comment:
can we remove the `RBENV_VERSION` there? If not, perhaps we can just specify `2.3` or even `2` - otherwise we will be depending on the CI image always having a 2.3.6 release.

Reply:
Is there any way to disable rbenv or use default version? Otherwise, using another image could work, like `centos/ruby-23-centos7`

Reply:
also, using a different image would work but would not be practical as another image would need to be download just for this and you would still encode the version here instead of using the app/ci image version

Reply:
used `.ruby-version` of the project :+1: